pkg/cryptutil/certificates.go

```diff
@@ -37,6 +38,24 @@ func CertificateFromFile(certFile, keyFile string) (*tls.Certificate, error) {
 	return &cert, err
 }
 
+// CRLFromBase64 parses a certificate revocation list from a base64 encoded blob.
+func CRLFromBase64(rawCRL string) (*pkix.CertificateList, error) {
+	bs, err := base64.StdEncoding.DecodeString(rawCRL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode base64 crl: %w", err)
```

Comment:
is it standard to add package name to error text? I think each error should just adequately identify the line where it all happened, and it would be further annotated by the calling code.